app/src/main/kotlin/no/nav/tiltakspenger/vedtak/routes/søknad/SøknadRoutes.kt

```diff
@@ -10,24 +10,32 @@ import no.nav.tiltakspenger.felles.Systembruker
 import no.nav.tiltakspenger.libs.auth.core.TokenService
 import no.nav.tiltakspenger.libs.auth.ktor.withSystembruker
 import no.nav.tiltakspenger.libs.soknad.SøknadDTO
+import no.nav.tiltakspenger.saksbehandling.domene.sak.Saksnummer
 import no.nav.tiltakspenger.saksbehandling.service.SøknadService
+import no.nav.tiltakspenger.saksbehandling.service.sak.SakService
 
 private val logger = KotlinLogging.logger {}
 
 const val SØKNAD_PATH = "/soknad"
 
-fun Route.søknadRoutes(søknadService: SøknadService, tokenService: TokenService) {
+fun Route.søknadRoutes(
+    søknadService: SøknadService,
+    sakService: SakService,
+    tokenService: TokenService,
+) {
     post(SØKNAD_PATH) {
         logger.debug { "Mottatt ny søknad på '$SØKNAD_PATH' -  Prøver deserialisere og lagre." }
         call.withSystembruker(tokenService = tokenService) { systembruker: Systembruker ->
             val søknadDTO = call.receive<SøknadDTO>()
             logger.debug { "Deserialisert søknad OK med id ${søknadDTO.søknadId}" }
-            // Oppretter sak med søknad og lagrer den
+            val sakId = sakService.hentForSaksnummer(Saksnummer(søknadDTO.saksnummer), systembruker).id
```

Comment:
Blir det loop hvis vi flytter dette kallet inn i SøknadService? Jeg synes det er ryddigest og bare gjøre et service kall.

Reply:
Etter at jeg gjorde saksnummer og sakid til en del av søknaden må jeg ha saken for å kunne opprette søknaden :/ Da må jeg i så fall dra med meg dto-formatet inn i søknadservicen og gjøre mappingen der.